src/assets/js/display-levels.js

```diff
@@ -15,6 +15,11 @@ export const DISPLAY_LEVELS = {
     icon: 'activity',
   },
   ndaed: {
+    label: "NDA'd",
+    value: 'NDAED',
+    icon: 'triangle',
+  },
+  ndaedplusstaff: {
```

Comment:
I think this duplication of `ndaed`/`ndaedplusstaff` is the wrong direction to go in. If we *really* want the "view as" drop-down to show something different from the "display level" drop-downs (which I don't think we do, as they're directly linked, and I hope we're reaching that consensus on JIRA), then I think we should just hard-code it here: https://github.com/mozilla-iam/dino-park-front-end/pull/479/commits/d5c531461399fb8c1c846302e393a8c3f1c072ab#diff-056eb55a41bbf1a5f9380763130b1732R91

Reply:
Yea, that makes good sense. Thanks for that catch!